app/styles/variables.less

```diff
@@ -1,3 +1,4 @@
 @navbar-height: 60px;
 @landing-offset-top: (@landing-search-area-height + @navbar-height);
 @oli-font-path: "../fonts";
+@font-size-base: 13px;
```

Comment:
Roll back this change because of https://github.com/openshift/origin-web-console/pull/2540 ?

Reply:
done.